examples/src/app.module.css

```diff
@@ -0,0 +1,78 @@
+html {
+  font-family: 'Roboto', sans-serif;
```

Comment:
I would go for system font stack as it doesn't force users to download custom fonts and does not promote any single design language as they vary between platforms.